playground/mocks/data/idp/idx/admin-consent.json

```diff
@@ -0,0 +1,99 @@
+{
+  "stateHandle": "02OCl7uyAUC4CUqHsObI9bvFiq01cRFgbnpJQ1bz82",
+  "expiresAt": "2018-09-17T23:08:56.000Z",
+  "version": "1.0.0",
+  "intent": "login",
+  "remediation": {
+    "type": "array",
+    "value": [{
+        "rel": [
+          "create-form"
+        ],
+        "name": "admin-consent",
+        "href": "http://localhost:3000/idp/idx/consent",
+        "method": "POST",
+        "accepts": "application/ion+json; okta-version=1.0.0",
+        "produces": "application/ion+json; okta-version=1.0.0",
+        "relatesTo": ["$.termsOfService", "$.privacyPolicy", "$.logo"],
+        "value": [
+        {
+          "name": "scopes",
+          "type": "object",
```

Comment:
I'm guessing it's probably making more sense as `array` as we send back list of scopes in request body (hence need to have some code changes in transformer layer)

Let's see what Shantanu get back to us.

Reply:
I asked Shantanu to change it to this, so that's what we'll get back

Reply:
It looks like other instances of `options` either have the type `object` or no type, so `type: array` would be a new inconsistency.

Reply:
iirc, the `type` dictates request body. default to `string` when it's missing.

Reply:
After reading about the options field, it looks like they're usually paired with type array: https://ionspec.org/#form-field-option-members
There'll be a design review of this response so I'll bring this up then

Reply:
It'll be changed to `array` so I'll handle that new type in uiSchemaTransformer in an upcoming commit